### PR TITLE
Curriculum landing page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-6732#

### CODE REVIEW

File: source/features/curriculum.html.erb [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  </div>
+
+  <div class="grey">
+    <div class="curriculum-img">
+      <%= image_tag "assets/features/curriculum/courses_mapped.png", class: "curriculum-content-img" %>
+    </div>
+  </div>
+
+  <div class='section top'>
+    <h1 class="">
```
Comment (mloginov) [inline source/features/curriculum.html.erb]: remove class attribute

File: source/features/curriculum.html.erb [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    <%= link_to 'Start FREE Trial', '/p/free-education-trial', class: 'button button-xlarge button-main' %>
+    <%= link_to 'Book a Demo', '/demo', class: 'button button-xlarge button-main button-demo' %>
+  </div>
+  <div class="clear">
+  </div>
+</section>
+
+<section class="curriculum-body">
+  <div class='section top'>
+    <h1 class="">
```
Comment (mloginov) [inline source/features/curriculum.html.erb]: remove class attribute

File: source/features/curriculum.html.erb [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    </p>
+  </div>
+
+  <div class="grey">
+    <div class="curriculum-img">
+      <%= image_tag "assets/features/curriculum/courses_mapped.png", class: "curriculum-content-img" %>
+    </div>
+  </div>
+
+  <div class='section top'>
```
Comment (mloginov) [inline source/features/curriculum.html.erb]: we use double quotes - fix it

File: source/features/curriculum.html.erb [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  <div class="action-buttons animated fadeInUp">
+    <%= link_to 'Start FREE Trial', '/p/free-education-trial', class: 'button button-xlarge button-main' %>
+    <%= link_to 'Book a Demo', '/demo', class: 'button button-xlarge button-main button-demo' %>
+  </div>
+  <div class="clear">
+  </div>
+</section>
+
+<section class="curriculum-body">
+  <div class='section top'>
```
Comment (mloginov) [inline source/features/curriculum.html.erb]: double quotes

**Comment by mloginov** [comment]
need merge with master

**Comment by denpira** [comment]
Done

**Comment by mloginov** [comment]
:+1: Ok for me

**Pull request merged**